### DISCUSSION TITLE
Publish to generic destinations using rclone

### DISCUSSION THREAD

**Discussion by ricklupton** [quarto-dev/quarto-cli#6931] [Q&A] [answered]
### Description

It would be useful to be able to publish my quarto documents to my own website, which can be accessed via WebDAV.

[rclone](https://rclone.org/) allows file upload/synching to many cloud services, including WebDAV.

If `quarto publish` had an rclone backend, it would automatically enable publishing to many other services without having to write a special case for each in quarto?

**Comment by mcanouil** [comment, marked as answer]
Note that meanwhile it might come in the future, you can use post-render script to achieve whatever you want for example using rclone.

`quarto publish` is not intended to be a comprehensive way to render and publish Quarto documents.
It is mostly an helper command which wrap `quarto render` and a post process script to publish.
Maintaining something that is not used is not really practical.
Users that used/know other services usually also know how to publish to them, thus they can be responsible of writing their publishing script which can be added as a post-render script in Quarto YAML.
